earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -120,22 +120,19 @@ void PacketGenAssetVersionImplD::UpdateChildren(
     prevTime = curTime;
 
     // Determine which insets intersect us at this level
-    const khLevelCoverage levCov(config.coverage.levelCoverage(level));
+    khLevelCoverage levCov(config.coverage.levelCoverage(level));
     std::vector<uint> neededIndexes;
+
     neededIndexes.reserve(numInsets);
-    if (type == AssetDefs::Imagery) {
-      FindNeededImageryInsets(khInsetCoverage(levCov),
-                              extra.insetInfo, numInsets,
-                              neededIndexes,
-                              config.beginMinifyLevel,
-                              config.endMinifyLevel);
-    } else {
-      FindNeededTerrainInsets(khInsetCoverage(levCov),
-                              extra.insetInfo, numInsets,
-                              neededIndexes,
-                              config.beginMinifyLevel,
-                              config.endMinifyLevel);
-    }
+    auto gencov = khInsetCoverage(levCov);
+    //overlapEnvelope<ProductAssetVersion> env(type, gencov,
+    //                     extra.insetInfo, extra.insetInfo.size(),
+    //                     config.beginMinifyLevel, config.endMinifyLevel, level);
+
+    extra.calc->env.gencov = gencov;
```

Comment:
Making a member variable public and modifying it external to the class is not a great pattern. It would be best to handle this another way. Maybe gencov shouldn't be part of the envelope but should be passed to the calls that need it.